pkg/ocp4/sdn/sdn_test.go

```
@@ -69,9 +69,9 @@ func TestGenYAML(t *testing.T) {
 	masterV3 := ocp3.Master{}
 	masterV3.Decode(content)
 
-	networkCR := Translate(masterV3.Config.NetworkConfig)
+	networkCR, _ := Translate(masterV3.Config.NetworkConfig)
```

Again don't hide the errors in tests. Use `t.Fatal` to fail the test